foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/databaseaccess/DatabasePlatform.java

```diff
@@ -2336,6 +2336,16 @@ public boolean supportsVPD() {
         return false;
     }
 
+    /**
+     *  INTERNAL:
+     *  Indicates whether the platform supports timeouts on For Update
+     *  
+     *  @see {@link DatabasePlatform#getSelectForUpdateWaitString(Integer waitTimeout)}
```

Comment:
link tag should not be required here

Reply:
I added it to better document the link between supportsWaitForUpdate() <-> getSelectForUpdateWaitString(Integer waitTimeout). If a platform wants supportsWaitForUpdate() to return true, then they need to override getSelectForUpdateWaitString(Integer waitTimeout) as well to make use of the wait amount.

Reply:
I'm OK with having `see` there, it is good thing. The point is that `see` should already generate the link itself (unless escaped by `:`), so one more `link` looks redundant

Reply:
... but I can be wrong here...

Reply:
And if you mean that `@link` is not needed, meaning `@see` is sufficient , I wanted it to be a clickable link

Reply:
Maybe the `@link` is needed if you are referencing a method in another class? I can't get THAT usecase to work... huh

Reply:
[docs](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDDIEDI) say that  `@see package.class#member label` should work; in this case just ` @see #getSelectForUpdateWaitString(Integer)` should be enough